controller/user.js

```diff
@@ -47,7 +50,7 @@ async function getByUsername(req, res) {
       code: 0,
       msg: "",
       data: {
-        user: buildUserJson(user)
+        user: formatUser(user)
```

Comment:
rename function to `getUserByUsername`

set `msg: "User"`, please refer to the API docs